indigo/indigo/src/main/scala/indigo/shared/formats/GenericTileset.scala

```diff
@@ -0,0 +1,39 @@
+package indigo.shared.formats
```

Comment:
An unexpected but good choice for the package. I was wondering where it should go after you suggested this feature! After you've looked into the comments below, please add an alias to the `package.scala` (copy the type+val pattern in there).

Reply:
OK. While I'm at it, I'll add them for Aseprite and TiledMap in the same package.

Reply:
Never mind, already there.